server/routes/severity-view.js

```diff
@@ -0,0 +1,28 @@
+const boom = require('@hapi/boom')
+const SeverityView = require('../models/severity-view')
+
+module.exports = {
+  method: 'GET',
+  path: '/severity-view',
+  options: {
+    ext: {
+      onPreAuth: {
+        method: (request, h) => {
+          // Set login redirect cookie
+          h.state('login-redirect', request.path)
+          return h.continue
+        }
+      }
+    },
```

Comment:
This is no longer necessary and should be removed. See https://github.com/DEFRA/fws-app/commit/5a74b601e0b806bbc9ef95c3ebb72322a74b0aa0

Reply:
changes made